modules/imgproc/src/color_lab.cpp

```diff
@@ -3567,7 +3567,7 @@ struct Luv2RGBinteger
 
         long long int xv = ((int)up)*(long long)vp;
         int x = (int)(xv/BASE);
-        x = y*x/BASE;
+        x = ((long long int)y)*x/BASE;
```

Comment:
Could you please add test for the case with overflow?
Existed `Imgproc_ColorLuv_Full.bitExactness` doesn't cover this code at all.

@savuor Which test should execute this "bit-exact" code (`Luv2RGBinteger`)?

Reply:
I tried to create a test but it is tough: a Luv(49,205,23) gives RGB(0,103,255) without my patch and (255,0,255) with my patch. Then again, converting back to Luv we respectively get (122,80,15) and (153,157,31). We hit the white point limitation as referenced in the doc and here https://answers.opencv.org/question/205330/converting-luv-rgb-luv-results-in-different-values/

If we use https://www.easyrgb.com/en/convert.php, the results are very sensitive at this point and due to rounding, I cannot tell which value should be returned.

I guess we hit the limits of the space and there is no real answer: https://upload.wikimedia.org/wikipedia/commons/8/8d/SRGB_gamut_within_CIELCHuv_color_space_mesh.webm

Reply:
Ok, I added a test to ensure it gives the same result as the float version.

Reply:
@alalek This is `Imgproc_ColorLuv_Full.bitExactness`, already fixed by @vrabaud

Reply:
@alalek The test you've mentioned should check this code for bit-exactness. If it's not true, then why?

Reply:
@savuor : the bit exactness code has the same overflow bug https://github.com/opencv/opencv/blob/d58b5ef74b91ba68ef89f9ce8be8725d0dfcaa0d/modules/imgproc/test/test_color.cpp#L2572 (that's why I had to modify it too). The fact that I had to change the hashes shows that there is indeed a bug.